system-test/fixtures/sample/src/index.js

```diff
@@ -16,12 +16,11 @@
 // ** https://github.com/googleapis/gapic-generator-typescript **
 // ** All changes to this file may be overwritten. **
 
-
 /* eslint-disable node/no-missing-require, no-unused-vars */
-const workflows = require('@google-cloud/workflows');
+const executions = require('@google-cloud/workflows');
 
 function main() {
-  const workflowsClient = new workflows.WorkflowsClient();
+  const executionsClient = new executions.ExecutionsClient();
```

Comment:
nit: in this test fixture we only create executions client. it might be OK, or we may want to exclude it (and its TypeScript counterpart) too and create both clients.

Reply:
I've switched the generation order so that this smoke test should create a `WorkflowsClient`. Let's wait for the nightly generation, and see if the smoke test looks reasonable.